gammapy/maps/wcsnd.py

```diff
@@ -468,7 +468,10 @@ def downsample(self, factor, preserve_counts=True):
 
     def plot(self, ax=None, fig=None, add_cbar=False, stretch='linear', **kwargs):
```

Comment:
If we only support interactive plotting of one map axes, I think we should add an `axes` argument to specify which axis should be plotted.

Reply:
For now the plot function only works for 3D Maps therefore there is no axes selection. I agree that we should implement that in the mid-term.